fe/fe-core/src/main/java/com/starrocks/load/loadv2/BrokerLoadJob.java

```diff
@@ -98,6 +104,37 @@ public void beginTxn()
                         timeoutSecond);
     }
 
+    @Override
+    public void alterJob(AlterLoadStmt stmt) throws DdlException {
+        writeLock();
+
+        try {
+            if (stmt.getAnalyzedJobProperties().containsKey(LoadStmt.PRIORITY)) {
+                priority = LoadPriority.priorityByName(stmt.getAnalyzedJobProperties().get(LoadStmt.PRIORITY));
+                AlterLoadJobOperationLog log = new AlterLoadJobOperationLog(id,
+                        stmt.getAnalyzedJobProperties());
+                GlobalStateMgr.getCurrentState().getEditLog().logAlterLoadJob(log);
+
+                for (LoadTask loadTask : newLoadingTasks) {
+                    GlobalStateMgr.getCurrentState().getLoadingLoadTaskScheduler().updatePriority(
+                            loadTask.getSignature(),
+                            priority);
+                }
+            }
+
+        } finally {
+            writeUnlock();
+        }
+        
+    }
+
+    @Override
+    public void replayAlterJob(AlterLoadJobOperationLog log) {
+        if (log.getJobProperties().containsKey(LoadStmt.PRIORITY)) {
```

Comment:
If the load job does not have the PRIORITY, will it be set to default `NORMAL` priority?

Reply:
yes